.gitignore

```diff
@@ -20,3 +20,6 @@ bin/*
 
 doc/
 .yardoc/
+
+.DS_Store
```

Comment:
fyi, for system-specific stuff you can create a global ~/.gitconfig file and then run

`git config --global core.excludesfile '~/.gitignore'`